circuits/lib/authenticationWithRelay.circom

```diff
@@ -1,64 +0,0 @@
-pragma circom 2.0.0;
-
-include "idOwnershipBySignatureWithRelay.circom";
-
-template VerifyAuthenticationWithRelay(IdOwnershipLevels, RelayLevels) {
-
-	signal input userClaimsTreeRoot;
-	signal input userAuthClaimMtp[IdOwnershipLevels];
-	signal input userAuthClaim[8];
-
-	signal input userRevTreeRoot;
```

Comment:
maybe we can keep it for now?

Reply:
Ok, I'll bring it back.

Reply:
@vmidyllic, after thinking a while I insist to do nothing with it at the moment. At least before the time the product managers decide whether we need it or not.
Up until that time, let's do nothing. We can return the circuits back from the history any time in the future.